src/core/Modal/Modal.tsx

```diff
@@ -170,10 +176,20 @@ export const Modal = (props: ModalProps) => {
 
   return !!container ? (
     ReactDOM.createPortal(
-      isOpen && (
-        <FocusTrap>
+      <FocusTrap>
+        <CSSTransition
+          in={isOpen}
+          classNames='iui-modal-animation'
+          timeout={4000}
```

Comment:
Change to `400` as it is in CSS animations.

Reply:
Whoops. Done